### PR TITLE
f-loyalty@v1.1.0 - Adding slots component

### PR DESCRIPTION
**Pull request by matthewhardern**
PR to add slots component for loyalty cards. Adds three areas for cards, first is redeemable cards, second is in progress stamps and third is recommended restaurants. Also there is a no cards error state when there are no cards. I have added unit tests for all the states. I will update with images. 

## UI Review Checks

Screenshot 2021-11-09 at 11 21 12
Screenshot 2021-11-09 at 11 20 49
Screenshot 2021-11-09 at 11 21 00

- [x] Unit tests have been [created|updated]

## Browsers Tested

- [x] Chrome (latest)

### CODE REVIEW

File: packages/components/pages/f-loyalty/src/components/Slots.vue [hunk 1]
```diff
@@ -0,0 +1,202 @@
+<template>
+    <div>
+        <div
+            v-if="!hasInProgressStampcards && !hasRedeemableStampcards"
+        >
```
Comment (minrwhite) [inline packages/components/pages/f-loyalty/src/components/Slots.vue]: Please can this be collapsed:
```suggestion
        <div v-if="!hasInProgressStampcards && !hasRedeemableStampcards">
```

File: packages/components/pages/f-loyalty/src/components/_tests/Slots.test.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    describe('Redeemable Stamp Cards', () => {
+        beforeEach(() => {
+            // Arrange
+            wrapper = mount(Slots, {
+                localVue,
+                propsData: {
+                    cards: MOCK_CARDS
+                },
+                ...wrapperOptions
+            });
```
Comment (minrwhite) [inline packages/components/pages/f-loyalty/src/components/_tests/Slots.test.js]: I think having the definition for this (and `zeroCardsWrapper`) in the `beforeEach()` doesn't make sense when not all tests relate to it.

I'd prefer if these were factored out as a `createWrapper(cards)` function definitions in-place of the `let` directives that allow you to be up-front in each test what state you're expecting for the behaviour.

File: packages/components/pages/f-loyalty/src/components/_tests/Slots.test.js [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+        });
+
+        it('should display the readyToClaimTitle when hasRedeemableStampcards is true', () => {
+            // Act
+            const title = wrapper.find('[data-test-id="section-readyToClaim-title"]');
```
Comment (minrwhite) [inline packages/components/pages/f-loyalty/src/components/_tests/Slots.test.js:56]: Gap for readability:
```suggestion
            const title = wrapper.find('[data-test-id="section-readyToClaim-title"]');

```

File: packages/components/pages/f-loyalty/src/components/_tests/Slots.test.js [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+            });
+        });
+
+        it('should display if hasInProgressStampcards is false and hasRedeemableStampcards is false', () => {
+            // Act
+            const noCardsErrorState = zeroCardsWrapper.findAllComponents(NoCardsErrorState);
+
+            // Assert
+            expect(noCardsErrorState.exists()).toBe(true);
+        });
```
Comment (minrwhite) [inline packages/components/pages/f-loyalty/src/components/_tests/Slots.test.js:184]: Another gap:
```suggestion
        });

```

File: packages/components/pages/f-loyalty/src/components/_tests/Slots.test.js (hunk 2 above)
Comment (matthewhardern) [reply packages/components/pages/f-loyalty/src/components/_tests/Slots.test.js]: yea that makes sense good shout

**Review by minrwhite — DISMISSED** [review]

File: packages/components/pages/f-loyalty/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v1.3.0
+------------------------------
+*November 08, 2021*
```
Comment (simonsandersje) [inline packages/components/pages/f-loyalty/CHANGELOG.md]: Date will need updating

**Review by simonsandersje — DISMISSED** [review]

**Review by minrwhite — DISMISSED** [review]

**Review by minrwhite — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

**Pull request merged**